### ISSUE TITLE
Some effects reducing volume significantly

### ISSUE THREAD

**Issue by tuarrep** [Pittvandewitt/Wavelet#48]
Adding some effects reduces the volume significantly.

Adding a gain effect should allow compensating the volume reduction by increasing it a bit back.

**Comment by Pittvandewitt** [comment]
Could you please elaborate which effects are reducing volume significantly?
I would also like to know what device you are using, what music player you use, if you have legacy mode enabled or not and steps to reproduce the issue.

**Comment by tuarrep** [comment]
- All effects are reducing volume
- Pixel 3a
- Deezer as music player
- Not on legacy mode

To reproduce I just toggle equalizer in Deezer and the volume changes
significantly

>

**Comment by locuturus** [comment]
I want to report sort of similar behavior using YT Music on a Pixel 4a. Latest Pixel software. If I turn legacy mode OFF and use some combinations of Equalization and Effects the volume  will drop so low that I used to think it didn't work at all, but I can achieve a normal listening level (32 ohm earbuds) with volume just a bit under maximum. With legacy mode ON I wouldn't go much above 15% in a quiet room.

For additional clarity any one Equalization or Effects feature may be enabled in isolation without volume problems. But various combinations cause the issue. I would need to build a spreadsheet to capture all the combinations I've noticed while typing this. Sometimes the order that I enable features appears to matter, sometimes it's not apparent to me the feature is working. It's kind of a mess. Equalization options seem to be robust, but the Effects options are super janky for me.
